src/modules/m_stripcolor.cpp

```diff
@@ -83,6 +83,24 @@ class ModuleStripColor : public Module
 		return MOD_RES_PASSTHRU;
 	}
 
+	void OnUserPart(Membership *memb, std::string &partmessage, CUList &except_list)
+	{
+		User *user = memb->user;
+		Channel *channel = memb->chan;
+
+		if (!IS_LOCAL(user))
+			return;
+
+		bool active = (user->IsModeSet(usc) || channel->GetExtBanStatus(user, 'S').check(!user->IsModeSet(csc)))
```

Comment:
Checking `user->IsModeSet()` here is undesirable. It will prevent any user with the stripcolor mode set to have colors in their part message, even on channels that doesn't have the stripcolor mode set.